libs/amount/amount.hpp

```diff
@@ -116,6 +118,7 @@ namespace amount {
      * @return
      */
     int compareTo(const Amount& other) const;
+    int ipow(int base, int exp) const;
```

Comment:
definitely shouldn't be a class function

Reply:
It is private. But ok, I can leave it only in cpp file